src/clang/lib_clang.cr

```diff
@@ -5,7 +5,7 @@ require "./enums"
   @[Link(ldflags: "-L`xcode-select --print-path`/usr/lib")]
   @[Link(ldflags: "-rpath `xcode-select --print-path`/usr/lib")]
 {% else %}
-  @[Link(ldflags: "`llvm-config-3.6 --ldflags 2>/dev/null || llvm-config-3.5 --ldflags 2>/dev/null || llvm-config --ldflags 2>/dev/null`")]
+  @[Link(ldflags: "`llvm-config --ldflags 2>/dev/null`")]
```

Comment:
Maybe allow LLVM_CONFIG environment variable, so we can more easily target a LLVM version, instead of tweaking the default.

Reply:
Good idea ! I'm on it :)